config/default.js

```diff
@@ -1,15 +1,16 @@
-var apiPort = process.env.NODE_PORT || 3000; // eslint-disable-line no-var
-var apiUrl = 'http://localhost:' + apiPort; // eslint-disable-line no-var
-var frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+const apiPort = process.env.NODE_PORT || 3000; // eslint-disable-line no-var
+const apiUrl = `http://localhost:${apiPort}`; // eslint-disable-line no-var
+const frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+const allowOrigin = [frontendUrl, apiUrl];
```

Comment:
IMO array variable should use plural form and because this project is still alpha we can change it to a better name (fe `allowedOrigins`). What do you think of it?